Tools/machines/polaris-alcf/polaris_gpu_warpx.profile.example

```diff
@@ -0,0 +1,51 @@
+# Set the project name
+export proj=""  # change me!
+
+# swap to the Milan cray package
+# module swap craype-x86-rome craype-x86-milan
+
+# required dependencies
+module load cmake/3.23.2
+module load cudatoolkit-standalone
+
+# optional: for QED support with detailed tables
+# module load boost/1.81.0
+
+# optional: for openPMD and PSATD+RZ support
+module load cray-hdf5-parallel/1.12.2.3
+export CMAKE_PREFIX_PATH=/home/${USER}/sw/polaris/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=/home/${USER}/sw/polaris/gpu/adios2-2.8.3:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=/home/${USER}/sw/polaris/gpu/blaspp-master:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=/home/${USER}/sw/polaris/gpu/lapackpp-master:$CMAKE_PREFIX_PATH
+
+export LD_LIBRARY_PATH=/home/${USER}/sw/polaris/gpu/c-blosc-1.21.1/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/home/${USER}/sw/polaris/gpu/adios2-2.8.3/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/home/${USER}/sw/polaris/gpu/blaspp-master/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/home/${USER}/sw/polaris/gpu/lapackpp-master/lib64:$LD_LIBRARY_PATH
+
+export PATH=/home/${USER}/sw/polaris/gpu/adios2-2.8.3/bin:${PATH}
+
+# optional: for Python bindings or libEnsemble
+module load cray-python/3.9.13.1
+
+if [ -d "/home/${USER}/sw/polaris/gpu/venvs/warpx" ]
+then
+  source /home/${USER}/sw/polaris/gpu/venvs/warpx/bin/activate
+fi
+
+# necessary to use CUDA-Aware MPI and run a job
+export CRAY_ACCEL_TARGET=nvidia80
+
+# optimize CUDA compilation for A100
+export AMREX_CUDA_ARCH=8.0
+
+# optimize CPU microarchitecture for AMD EPYC 3rd Gen (Milan/Zen3)
+# note: the cc/CC/ftn wrappers below add those
+# export CXXFLAGS="-march=znver3"
+# export CFLAGS="-march=znver3"
```

Comment:
These lines are commented for now since apparently the available `nvhpc/21.9` compilers do not like these flags. See "Notes on Default Modules" at https://docs.alcf.anl.gov/polaris/compiling-and-linking/compiling-and-linking-overview/.

Reply:
@roelof-groenewald you might need to pass them as `-XCompiler="-march=znver3"` or they have a different syntax (NVHPC are the prior PGI compilers after all).

Reply:
I should be able to switch these environment variables on again with the switch to GNU. I'll test to confirm.